tests/Fixtures/project/src/Aspect/InitializationAspect.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Go\Tests\TestProject\Aspect;
+
+use Go\Aop\Aspect;
+use Go\Lang\Annotation as Pointcut;
+
+class InitializationAspect implements Aspect
+{
+    /**
+     * Before class instance initialization.
+     *
+     * @Pointcut\Before("initialization(Go\Tests\TestProject\Application\Main)")
```

Comment:
It's fragile joinpoint, maybe we shouldn't use it in framework at all? And just leave it as experimental feature with special configuration?